### PR TITLE
Add blocking indicator for activation request.

### PR DESCRIPTION
**Pull request by rabbah**
This allows non blocking requests to drop their results from the activeack response.

### CODE REVIEW

File: common/scala/src/main/scala/whisk/core/entity/WhiskActivation.scala [hunk 1]
```diff
@@ -90,6 +90,7 @@ case class WhiskActivation(
         }
     }
 
+    def withoutLogsOrResponse = copy(response = ActivationResponse.success(), logs = ActivationLogs()).revision[WhiskActivation](rev)
```
Comment (markusthoemmes) [inline common/scala/src/main/scala/whisk/core/entity/WhiskActivation.scala]: Falling back to a success always breaks the health-protocol doesn't it? We can safely drop the user's response stuff, but we need to send the correct statuscode I think.

**Comment by rabbah** [comment]
PG2/2027 🚀

**Comment by rabbah** [comment]
PG2/2027 🔵

**Comment by rabbah** [comment]
Squash all - thanks for asking.

**Pull request merged**